editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -1548,19 +1535,12 @@ export var storyboard = (
           await selectComponentsForTest(editor, [targetPath])
           await pressKey('c', { modifiers: cmdModifier })
 
-          const canvasRoot = editor.renderedDOM.getByTestId('canvas-root')
-
           FOR_TESTS_setNextGeneratedUids(['aaa', 'bbb', 'ccc', 'ddd'])
```

Comment:
do you know why `FOR_TESTS_setNextGeneratedUids` is unreliable here?

Reply:
I couldn't find out, so I removed it in https://github.com/concrete-utopia/utopia/pull/3798/commits/ab9ca56df93b493103d81604f97fc36fddaec1cf so that it doesn't cause any confusion